docs-source/content/userguide/managing-fmw-domains/soa-suite/WebLogic-Logging-Exporter-Setup.md

```diff
@@ -0,0 +1,133 @@
+# WebLogic Logging Exporter Setup to Publish Logs into Elasticsearch
+
+The WebLogic Logging Exporter adds a log event handler to WebLogic Server. 
+
+WebLogic Server logs can be pushed into Elasticsearch in Kubernetes directly, by using the Elasticsearch REST API. For more details refer to [weblogic-logging-exporter](https://github.com/oracle/weblogic-logging-exporter) project.  
+
+This documents provides the steps to publish WebLogic Server logs into Elasticsearch and interact with them in Kibana. 
+
+## Prerequisite
+
+This document assumes that you have already setup Elasticsearch/Kibana for logs collection. If you have not, please refer this [document](/kubernetes/samples/scripts/elasticsearch-and-kibana/README.md) to setup Elasticsearch/Kibana from WebLogic operator samples.
+
+---  
+
+Please follow the below steps to push the server logs into Elasticsearch.  
+
+## Download WebLogic Logging Exporter Binaries
+
+The WebLogic Logging Exporter pre-built binaries are available in the release page of the [weblogic-logging-exporter](https://github.com/oracle/weblogic-logging-exporter/releases) project.  
+You need to download the 
+* `weblogic-logging-exporter-0.1.1.jar` from this [link](https://github.com/oracle/weblogic-logging-exporter/releases)
+* `snakeyaml-1.23.jar` from [Maven Central](https://search.maven.org/artifact/org.yaml/snakeyaml/1.23/bundle).  
+
+## Copy downloaded JAR Files to the WebLogic Domain Home
+
+```
+NOTE: Please note that below are the identifiers used in the sample commands in this document.
+
+* soans: is the SOA Domain namespace.
+* soainfra: is the domainUID
+* soainfra-adminserver: is the Administration Server pod name
+```
+
+Copy the `weblogic-logging-exporter-0.1.1.jar` and `snakeyaml-1.23.jar` files to the domain home folder in the Administration Server pod.
+
+```
+Command:
+$ kubectl cp <file-to-copy> <namespace>/<Administration-Server-pod>:<domainhome>
+
+```
+
+```
+Sample commands: 
+$ kubectl cp snakeyaml-1.23.jar soans/soainfra-adminserver:/u01/oracle/user_projects/domains/soainfra/
+ 
+$ kubectl cp weblogic-logging-exporter-0.1.1.jar soans/soainfra-adminserver:/u01/oracle/user_projects/domains/soainfra/
+```
+
+## Add a Startup Class to the Domain Configuration
+
+In the Administration Console, navigate to "Environment" then "Startup and Shutdown classes" in the main menu.  
+Add a new Startup class. You may choose any descriptive name and the class name must be "weblogic.logging.exporter.Startup".  
+
+![Add startup class](images/WebLogic-Logging-Exporter_add-startup-class.png)  
+
+Target the startup class to each server that you want to export logs from.  
```

Comment:
Target the startup class to each server that you want to export logs from. -> Target the startup class to each server from which you want to export logs.